AirCasting/SessionViews/SessionTime.swift

```diff
@@ -0,0 +1,48 @@
+// Created by Lunar on 02/12/2022.
+//
+
+import Foundation
+import SwiftUI
+
+struct SessionTime: View {
+    @ObservedObject var session: SessionEntity
```

Comment:
a simple ViewModel would be nice to handle this text formatting (but not really required since this file is just 50LOC)